### PR TITLE
[Support new order funnel - Part 4] Allowing ordering for all subsidiaries

### PR DESCRIPTION
**Pull request by FredericEspiau**
# Allowing ordering for all subsidiaries
This PR is part of the **Support new order funnel** project
## Part 4 - Allowing ordering for all subsidiaries
It looks like there is a lot of changes but every commit contains only a few character changes.
If you remove whitespaces while reviewing, then you will see that not a lot have changed.

### :rocket: Features
* The option order funnel is available to all subsidiaries

### :nail_care: Refactor
* Many files have been renamed to tell apart legacy order and current order funnel

### :house: Internal
* Bumped the version of `ui-router` to fix an issue with view targeting

## References
- MANAGER-2372

### CODE REVIEW

File: client/app/dedicatedCloud/dedicatedCloud.routes.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ angular
   .module('App')
   .config(/* @ngInject */ ($stateProvider, $urlServiceProvider) => {
     $stateProvider.state('app.dedicatedClouds', {
+      reloadOnSearch: false,
```
Comment (marie-j) [inline client/app/dedicatedCloud/dedicatedCloud.routes.js]: Imo it's easier to have the `url` first in terms of reading even though alphabetical order seems logical as you don't always have the same properties (you can have `controller`, `views`, `layout` ...)

File: client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.routes.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ angular
   .config(($stateProvider, constants) => {
     if (constants.target === 'US') {
       $stateProvider.state('app.dedicatedClouds.license.enable', {
-        url: '/enable',
-        templateUrl: 'dedicatedCloud/license/enable/dedicatedCloud-license-enable-us.html',
         controller: 'DedicatedCloudLicencesSplaEnableUSCtrl',
         controllerAs: '$ctrl',
         layout: 'modal',
+        templateUrl: 'dedicatedCloud/license/enable/dedicatedCloud-license-enable-us.html',
+        url: '/enable',
```
[the remark this answers is not in the document]
Comment (marie-j) [reply client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable.routes.js]: As discusses, nevermind about my previous comment :)

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**